ads/google/a4a/utils.js

```diff
@@ -480,3 +487,28 @@ export function extractAmpAnalyticsConfig(
   }
   return null;
 }
+
+/**
+ * Add new experiment IDs to a (possibly empty) existing set of experiment IDs.
+ * The {@code currentIdString} may be {@code null} or {@code ''}, but if it is
+ * populated, it must contain a comma-separated list of integer experiment IDs
+ * (per {@code parseExperimentIds()}).  Returns the new set of IDs, encoded
+ * as a comma-separated list.  Does not de-duplicate ID entries.
+ *
+ * @param {!string} newIds IDs to merge in. Must be a a string containing a
+ *   comma-separated list of stringified integer (base 10) experiment IDs.
+ * @param {?string} currentIdString  If present, a string containing a
+ *   comma-separated list of integer experiment IDs.
+ * @returns {string}  New experiment list string, including newId iff it is
+ *   a valid (integer) experiment ID.
+ * @see parseExperimentIds, validateExperimentIds
+ */
+export function mergeExperimentIds(newIds, currentIdString) {
+  const newIdArr = newIds.split(',').filter(newId => {
+    return newId && !isNaN(parseInt(newId, 10));
```

Comment:
The standard appears to be to wrap with Number constructor and I assume we're ok with excluding 0.  In addition you can just use join so this can be:

return newIds.split(',').filter(newId => Number(newId)).join(',')